configuration.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I don't think we need to create `.md` files just to include links to other documents. You can just add the links to the ToC or make this a single document.